ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/ScanResultsOverviewTable.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import {
+    TableComposable,
+    Tbody,
+    Td,
+    Th,
+    Thead,
+    Tr,
+    ActionsColumn,
+    IAction,
+} from '@patternfly/react-table';
+import { format } from 'date-fns';
+
+import usePagination from 'hooks/patternfly/usePagination';
+import useTableSort from 'hooks/patternfly/useTableSort';
+
+import { ComplianceScanResultsOverview } from './types';
+import ScanResultsToolbar, { SearchFilter } from './ScanResultsToolbar';
+
+import scanResultsOverviewMockData from './MockData/complianceScanResultsOverview.json';
+
+const defaultSearchFilter = {
+    scanName: '',
+    clusterName: '',
+    profileName: '',
+};
+
+const sortFields = ['Scan Name', 'Failing Controls', 'Last Scanned'];
+const defaultSortOption = { field: 'Last Scanned', direction: 'asc' } as const;
+
+function ScanResultsOverviewTable() {
+    const { page, perPage, onSetPage, onPerPageSelect } = usePagination();
+    const { sortOption, getSortParams } = useTableSort({ sortFields, defaultSortOption });
+    const [searchFilter, setSearchFilter] = useState<SearchFilter>(defaultSearchFilter);
+    const [scanResultsOverviewData] = useState<ComplianceScanResultsOverview[]>(
+        scanResultsOverviewMockData.scanOverviews
+    );
+
+    useEffect(() => {
+        // TODO: fetch GetComplianceScanResultsOverview once implemented and update scanResultsOverviewData
+    }, [searchFilter, page, perPage, sortOption]);
```

Comment:
If this is going to be a relatively normal, single REST endpoint call, you can [`useRestQuery()` ](https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/hooks/useRestQuery.ts) to get rid of the effect and get loading/error handling for free. The one caveat is that you would need to wrap the service function in a useCallback like https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Collections/CollectionsTablePage.tsx#L49